pkg/controller/statusmanager/status_manager.go

```diff
@@ -3,21 +3,22 @@ package statusmanager
 import (
 	"context"
 	"fmt"
-	"k8s.io/apimachinery/pkg/labels"
 	"log"
 	"os"
 	"reflect"
 	"strings"
 	"sync"
 
+	"github.com/openshift/cluster-network-operator/pkg/platform"
+	"k8s.io/apimachinery/pkg/labels"
```

Comment:
nit: the imports in this file are already all over the place but the platform immport could go where the network import was and the apimachinery import could go in the apimachinery import group.

Reply:
good point, sorry for missing the comment sooner